Diana's Files/Columbia.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<title>W3.CSS Template</title>
```

Comment:
Change all title tags for all the pages to "BAResources" or maybe "Columbia | BAResources" (the page name and have the site name following).